### DISCUSSION TITLE
Using RS256 algo in the JWT

### DISCUSSION THREAD

**Discussion by awnish-npg** [directus/directus#7663] [Archive]
We want to use the RS256 in the application so please suggest the location of the private key and public key files which can be used in the RS256 token signature generation.

**Comment by rijkvanzanten** [comment]
You can't currently override the options used for directus' jwt generation. (ref https://github.com/directus/directus/discussions/7670)

**Reply by awnish-npg** [reply]
Okay. I understand. I have found the way to update it using the options in the ENV by including the public and private key path in there. Should I push the change in my branch for a review and if required this can be released in next milestone.

**Comment by spatialy** [comment]
Hi @awnish-npg, anywhere to see your implementation?
